app/models/course.rb

```diff
@@ -0,0 +1,4 @@
+# Class describing a course
+class Course < ApplicationRecord
+  validates_presence_of :name, presence: true, uniqueness: true
```

Comment:
It would be better to have two distinct validations here instead, `validates_presence_of` and `validates_uniqueness_of`. 

You also don't need to specify `presence: true` for the validates_presence_of validator

Reply:
Let's also add a validation to restrict the name of the course. We can use the same validation that we use on the assessment short_identifier: 

https://github.com/MarkUsProject/Markus/blob/0519dec2aa024b880db86ebbdf690cae0f324d65/app/models/assessment.rb#L23-L25